### PR TITLE
Unity Avatar Exporter Improvements

### PR DESCRIPTION
**Pull request by SaracenOne**
Ongoing collection of commits to fix bugs and add new features to the Unity avatar exporter.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19828/HighFidelity-Beta-PR15701-4266928.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19828/HighFidelity-Beta-Interface-PR15701-4266928.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19828/HighFidelity-Beta-PR15701-4266928.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19828/HighFidelity-Quest-Beta-PR15701-4266928.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19828/HighFidelity-Beta-PR15701-4266928.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19828/HighFidelity-Beta-Interface-PR15701-4266928.exe

File: tools/unity-avatar-exporter/Assets/Editor/AvatarExporter/AvatarExporter.cs [hunk 1]
```diff
@@ -645,10 +660,13 @@ class MaterialData {
     }
     
     static bool WriteFST(string exportFstPath, string projectName, float scale) {        
-        // write out core fields to top of fst file
+		// Calculate the actual scale from the model import settings
+        float finalScale = scale * modelImporter.globalScale;
```
Comment (huffman) [inline tools/unity-avatar-exporter/Assets/Editor/AvatarExporter/AvatarExporter.cs]: Hi @SaracenOne! Thanks for submitting these improvements to the exporter. I'm trying to grok this change here - in what case would the globalScale not be 1 (maybe if an FBX isn't using cm, for example?) ? Any other elaboration would be helpful as well!

**Comment by shanzzam** [comment]
@SaracenOne Feedback requested

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by ZappoMan** [comment]
We're going to close this PR because it hasn't seen any action since July. Please feel free to reopen it with a more clear explanation of what the change is fixing, along with example content the exercises the change. Thanks!

**Pull request closed without merging**